src/data/bool.lean

```diff
@@ -112,10 +112,11 @@ theorem eq_tt_of_bnot_eq_ff : ∀ {a : bool}, bnot a = ff → a = tt := dec_triv
 theorem eq_ff_of_bnot_eq_tt : ∀ {a : bool}, bnot a = tt → a = ff := dec_trivial
 
 @[simp] theorem bxor_comm : ∀ a b, bxor a b = bxor b a := dec_trivial
```

Comment:
I think this should not be a `simp` lemma

Reply:
I agree.
I guess I could remove it as part of this PR, and hope it doesn't break to much.